### PR TITLE
[Headless Chrome] Update dev requirements of spiritix/php-chrome-html2pdf for 1.6 version

### LINKED ISSUE #12299: [Bug]: Error during PDF-Generation:Notice: fwrite(): Write of 156826 bytes failed with errno=32 Datenübergabe unterbrochen (broken pipe)
### Expected behavior

Generate a PDF Document

### Actual behavior

Error during PDF-Generation:Notice: fwrite(): Write of 156826 bytes failed with errno=32 Datenübergabe unterbrochen (broken pipe) after Update to PIMCORE 10.4.2 from 10.4.1 yesterday

After the second click on the "Gernerate Button" this warning occurs:

Error during PDF-Generation:Binary error: /home/xxx/xxx/vendor/spiritix/php-chrome-html2pdf/index.js:11
import getStdin from 'get-stdin';
       ^^^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:760:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
    at internal/main/run_main_module.js:21:11
 

### Steps to reproduce

generate an PDF Document

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #12299

### CODE REVIEW

File: doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ Additionally, two settings can be used to add header and footer by the content o
 * header: URL for header content
 * footer: URL for footer content
 
+> Note: Only package spiritix/php-chrome-html2pdf:1.6.x is supported, not higher version due to dependency on Node > 18.
+
```
Comment (NiklasBr) [inline doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md:99]: Why not? We are currently using Node 18 just fine for Chrome Headless

Comment (dvesh3) [reply doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md:99]: yes it works but then pimcore bundles like portal engine or statistics explorer are not supported on Node 18. There could be more side effects.

Comment (NiklasBr) [reply doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md:99]: Seems like a a potential security issue to revert back to abandoned versions of Puppeteer, Node, and NPM though, no? If those specific bundles require older versions, why not specify in those composer files that they have specific requirements?

Comment (brusch) [reply doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md:99]: Agree - closed, it's not a `pimcore/pimcore` issue. Thanks for your feedback.

**Pull request closed without merging**